tests/manage/cluster/cluster_shutdown/test_shutdown.py

```diff
@@ -0,0 +1,105 @@
+import logging
+import pytest
+
+from ocs_ci.framework import config
+from ocs_ci.utility.utils import ceph_health_check
+from ocs_ci.ocs import ocp
+from ocs_ci.framework.testlib import tier4, ManageTest
+from ocs_ci.utility import aws
+from ocs_ci.ocs.cluster import CephCluster
+
+logger = logging.getLogger(__name__)
+
+
+@pytest.fixture()
+def instances(request):
+    """
+    Get cluster instances
+
+    Returns:
+        list: The cluster instances dictionaries
+    """
+    def finalizer():
+        """
+        Make sure all instances are running
+        """
+        aws.start_instances(instances)
+
+    request.addfinalizer(finalizer)
+
+    instances = ocp.get_all_nodes()
+    return instances
+
+
+@pytest.fixture()
+def vms():
+    # TODO
+    pass
+
+
+@pytest.fixture()
+def bm_machines():
+    pass
+    # TODO
+
+
+class BaseClusterShutdown(ManageTest):
+    """
+    Base class for cluster shutdown related tests
+    """
+    def validate_cluster(self):
+        assert ocp.wait_for_nodes_ready(len(instances)), (
+            "Not all nodes reached status Ready"
+        )
+
+        ceph_cluster = CephCluster()
+        assert ceph_health_check(
+            namespace=config.ENV_DATA['cluster_namespace']
+        )
+        ceph_cluster.cluster_health_check(timeout=60)
+
```

Comment:
@ebenahar as part of validate cluster, will it be good to have a small check for the folllowing:

1. able to create 1 PVC + POD for both RBD and Cephfs ( confiming that cephcluster is usable even after full cluster shutdown)

Please share your opinion.

Reply:
Usually the Tier 4 tests need some background operations to keep running throughout the test. As part of that, the above mentioned operation should be covered. Not sure cluster_health_check will sample the results of such background operations. Or if the background operations itself confirm the success of pvc, pod and IOs success, then no need to explicitly check that.

Reply:
@nehaberry yes, I'll add these checks after the cluster health check.
@hnallurv I think for that, we can execute this test with --io_in_bg that will run IO in the background. Although, I'm pretty sure we'll see ugly exceptions from the IO thread due to the shutdown. Will try that later